packages/editor/src/lib/utils/licensing.ts

```diff
@@ -0,0 +1,117 @@
+import crypto from 'crypto'
```

Comment:
ah, this is just for Jest testing - we shouldn't import `crypto` :)

Reply:
I get failed tests when I run them locally without importing crypto. Is that ok?

Reply:
yeah we just need to do `import crypto from 'crypto'` inside of `LicenseManager.test.ts` instead of in here.